src/lib/ByteConverters.ts

```diff
@@ -145,6 +147,15 @@ export const toBytesVector = <T extends ToBytes>(vec: T[]): Uint8Array => {
   return concat(valueByteList);
 };
 
+/**
+ * Use blake2b to compute hash of ByteArray
+ * @param x Byte array of type `Uint8Array` to compute the blake2b hash on
+ * @returns `Uint8Array` buffer of the blake2b hash
+ */
+export function byteHash(x: Uint8Array): Uint8Array {
+  return blake.blake2b(x, null, 32);
+}
+
```

Comment:
good move with shifting it to this file, you can also switch to use this method `byteHash` in `DeployUtil` file https://github.com/casper-ecosystem/casper-js-sdk/blob/eb5461bbc55fb9adedaa2cf5d819c464a38d38ae/src/lib/DeployUtil.ts#L1355 rather then manually invoking `blake.blake2b`